sql-cli/sql_cli/__main__.py

```diff
@@ -37,20 +38,19 @@ def about() -> None:
 
 @app.command(help="Generate the Airflow DAG from a directory of SQL files.")
 def generate(
-    directory: Path = typer.Argument(
+    workflow_directory: Path = typer.Argument(
         default=...,
         show_default=False,
         exists=True,
         help="directory containing the sql files.",
     ),
-    dags_directory: Path = typer.Argument(
```

Comment:
I believe, from the generate perspective, it is better to have the `dags_directory` - since Airflow users can use it as a stand-alone command and generate the DAG anywhere, they'd like - without the need to be respecting the SQL CLI project structure for output.

Reply:
I have removed it for now. I think we can re-add it later. I would not add to many arguments in the beginning especially because this as you said would only be useful to airflow users - not necessarily sql-cli users.